menas/ui/components/mappingTable/mappingTableMain.controller.js

```diff
@@ -321,6 +321,11 @@ sap.ui.controller("components.mappingTable.mappingTableMain", {
       sap.ui.getCore().byId("newMappingTableName").setValueStateText("Mapping Table name '" + oMT.name + "' already exists. Choose a different name.")
       isOk = false;
     }
+    if (/\W/.test(oMT.name)){
```

Comment:
Let's use all `if`s to find and highlight all validation errors in one go. If we return on the first validation error (or have `else`s), only it will be displayed.

Reply:
That's preferable for sure, but would require little more changes otherwise the later `if` overwrites the message of the earlier ones (`setValueStateText` on the same component)

Reply:
Right, should clarify (didn't peer into this too close): `if` per field with a series of `else`s for each validation on that field.